packages/sdk-client/src/builder/ClientBuilder.ts

```diff
@@ -64,7 +64,7 @@ export default class ClientBuilder {
       .withLoggerMiddleware()
   }
 
-  withAuthMiddleware(authMiddleware: Middleware): ClientBuilder {
+  private withAuthMiddleware(authMiddleware: Middleware): ClientBuilder {
```

Comment:
it's fine with this one as being public as it opens the possibility to add a custom AuthMiddleware

```suggestion
  withAuthMiddleware(authMiddleware: Middleware): ClientBuilder {
```

Reply:
Done